modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/property/SdkSourceFactoryModelProperty.java

```diff
@@ -16,32 +17,32 @@
  *
  * @since 4.0
  */
-public final class SourceFactoryModelProperty implements ModelProperty {
+public final class SdkSourceFactoryModelProperty implements ModelProperty {
```

Comment:
Why was it renamed?

Reply:
It was renamed because it now holds an SdkSourceFactory, which is a new interface defined here https://github.com/mulesoft/mule-extensions-api/pull/758/files .